LifeCycleAssessment_Engine/Compute/EvaluateEnvironmentalProductDeclaration.cs

```diff
@@ -46,6 +48,7 @@ public static LifeCycleAssessmentElementResult EvaluateEnvironmentalProductDecla
         {
             double value = 0;
             EnvironmentalMetricResult resultValue = null;
+            MaterialComposition mc = elementM.IMaterialComposition();
 
             List<QuantityType> qts = elementM.GetQuantityType();
```

Comment:
GetQuantityType here calls IMaterialComposition again. We need an overload method for GetQuantityType with an mc input similar to the others, and that method should be called here using mc above.